client/curriculum/html/AudioAndVideo.tsx

```diff
@@ -0,0 +1,345 @@
+import React from 'react';
+import SEOTags from '~/components/SEOTags';
+import Lesson, {
+  LessonContributors,
+  LessonCover,
+  LessonHeading,
+  LessonTip,
+  LessonResources,
+} from '~/components/lessons';
+import { SebastianLatkolic } from '~/services/contributors';
+import Highlight from '~/components/Highlight/Highlight';
+import coverSvg from '~/public/images/lessons/video_cover.svg';
+
+const contributors = [SebastianLatkolic];
+const chapters = [
+  { title: 'Elementul <audio>', id: 'audio-element' },
+  { title: 'Atributele elementului <audio>', id: 'atributele-elementului-audio' },
+  { title: 'Elementul <video>', id: 'video-element' },
+  { title: 'Atributele elementului <video>', id: 'atributele-elementului-video' },
+];
+
+export default function ListsLesson() {
+  return (
+    <>
+      <SEOTags
+        title="Audio și Video | Lecție HTML"
+        shareImage="https://frontend.ro/seo/html-video.jpg"
+        description="Învață despre elementele audio și video în HTML"
+        url="https://FrontEnd.ro/html/video-audio"
+      />
+      <Lesson title="Audio și Video" chapters={chapters}>
+        <LessonContributors className="absolute" contributors={contributors} />
+        <LessonCover>
+          {/* eslint-disable-next-line react/no-danger */}
+          <div dangerouslySetInnerHTML={{
+            __html: coverSvg,
+          }}
+          />
+        </LessonCover>
+        <p>
+          După cum ați văzut în lecția anterioară, putem adăuga imagini pe site-urile noastre. Dar cum ar fi dacă v-am spune că putem adăuga fișiere audio sau chiar video pentru a oferi și mai multă interactivitate paginilor dezvoltate de noi? Ei bine, hai sa vedem cum putem face asta.
+        </p>
+        <section>
+          <LessonHeading as="h3" id="audio-element">
+            {'Elementul <audio>'}
+          </LessonHeading>
+          <p>
+            Elementul
+            {' '}
+            <strong className="formatted">{'<audio>'}</strong>
+            {' '}
+            este folosit pentru a adăuga fișiere audio paginii tale web. Acesta se declară cu ajutorul atributului
+            {' '}
+            <strong className="formatted">{'src'}</strong>
+            .
+          </p>
+          <Highlight
```

Comment:
You should have some demos for every Highlight Component. Please see Lists lesson for example.

Reply:
@andreeatoma I'll need some audio and video files for that. Any suggestion? Maybe some custom ones.